packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -1013,9 +1009,9 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
   }
 
   void _sortData(
-    ColumnData<T> column,
+    ColumnData<T?> column,
```

Comment:
https://github.com/rrousselGit

Reply:
this looks like an accidental paste from when I was updating the triage doc. disregard.